sklearn/model_selection/_split.py

```diff
@@ -759,6 +763,8 @@ class TimeSeriesSplit(_BaseKFold):
     max_train_size : int, optional
         Maximum size for a single training set.
 
+        ..versionadded:: 0.19
```

Comment:
```suggestion
        .. versionadded:: 0.19
```

Reply:
This should be above, in the summary section